javagen/src/main/java/com/azure/autorest/template/ProxyTemplate.java

```diff
@@ -111,12 +112,17 @@ public final void write(Proxy restAPI, JavaClass classBlock) {
                                 break;
 
                             case Body:
+                                if (restAPIMethod.getRequestContentType() != null && restAPIMethod.getRequestContentType().equals(ContentType.APPLICATION_X_WWW_FORM_URLENCODED)) {
```

Comment:
Could invert this check and compact it:

```suggestion
                                if (ContentType.APPLICATION_X_WWW_FORM_URLENCODED).equals()restAPIMethod.getRequestContentType()) {
```